modules/openapi-generator/src/main/resources/dart-jaguar/model_test.mustache

```diff
@@ -0,0 +1,26 @@
+{{#models}}
+{{#model}}
+import 'package:{{pubName}}/api.dart';
+import 'package:test/test.dart';
+
+// tests for {{classname}}
+void main() {
+  var instance = new Pet();
```

Comment:
No longer builds. This should be
```dart
var instance = new {{classname}}();

Reply:
Good catch. I'll fix it.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/3014